### PR TITLE
ENT-5967 Correct release AIX version

### PR DESCRIPTION
**Pull request by Lex-2008**
----

#

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a PR! Maybe @craigcomstock can review this?

**Comment by Lex-2008** [comment]
test package installs fine!

File: packaging/cfengine-nova/cfengine-nova.bff.sh [hunk 1]
```diff
@@ -15,6 +15,13 @@ VER3=$(echo $VERSION | cut -d. -f3)
 VER4=$(echo $VERSION | cut -d. -f4)
 # In third field: change 'a' to '88', 'b' to '99', delete leading zeroes if followed by a number
 VER3=$(echo $VER3 | sed -e 's/a/88/;s/b/99/;s/^0\(.\)/\1/')
+# If third field contains '-' (happens during release builds) - remove it, adding stuff after dash to the beginning of 4th field.
+# i.e. from $VER3=1-2 $VER4=3 go to $VER3=1 $VER4=23
+# Note that currently, when third field contains '-', 4th field is empty
+if echo $VER3 | grep - >/dev/null; then
+    VER4="$(echo $VER3 | sed -e 's/.*-//')$VER4"
+    VER3="$(echo $VER3 | sed -e 's/-.*//')"
```
Comment (vpodzime) [inline packaging/cfengine-nova/cfengine-nova.bff.sh:23]: I'd prefer using `cut` like above.

Comment (Lex-2008) [reply packaging/cfengine-nova/cfengine-nova.bff.sh]: good point! Amended!

**Comment by craigcomstock** [comment]
This needs to go to 3.15.x and maybe a re-spin of 3.15.2 packages? Should also check 3.12.5 packages.

**Comment by Lex-2008** [comment]
3.15.x PR: https://github.com/cfengine/buildscripts/pull/752
3.12.x PR: https://github.com/cfengine/buildscripts/pull/753

**Pull request merged**